docs/presto-admin-commands.rst

```diff
@@ -489,14 +489,14 @@ server upgrade
 **************
 ::
 
-    presto-admin server upgrade local_package_path [local_config_dir]
+    presto-admin server upgrade new-rpm [local_config_dir]
 
-This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM specified
-by ``local_package_path``, preserving the existing configuration on the cluster. The existing
+This command upgrades the Presto RPM on all of the nodes in the cluster to the RPM
+``new-rpm``, preserving the existing configuration on the cluster. The existing
```

Comment:
Okay, let me rephrase it:
if we expect absolute path (e.g. because it's got to be available on all nodes in the same location or for whatever else reason), let's make it clear here.